source/MRViewer/ImGuiMenu.h

```diff
@@ -83,6 +83,15 @@ class MRVIEWER_CLASS ImGuiMenu : public MR::ViewerPlugin,
   std::weak_ptr<Object> lastRenameObj_;
   Box3f selectionBbox_; // updated in drawSelectionInformation_
 
+  struct LabelNameMemory
+  {
+      std::string lastLabel;
+      std::string labelBuffer;
+      Vector3f textPos;
+      std::shared_ptr<ObjectLabel> obj{ nullptr };
+      bool changed{ false };
+  } oldLabelName_;
```

Comment:
why save all, i think we need only label text, and why `oldLabelName_` if it holds all info